tests/test_computeExposureSummaryStats.py

```diff
@@ -102,10 +115,10 @@ def testComputeExposureSummary(self):
 
         # Configure and run the task
         expSummaryTask = ComputeExposureSummaryStatsTask()
-        # Configure nominal values for effective time calculation
-        expSummaryTask.config.fiducialZeroPoint = {band: float(zp)}
+        # Configure nominal values for effective time calculation (normalized to 1s exposure)
+        expSummaryTask.config.fiducialZeroPoint = {band: float(zp - 2.5*np.log10(expTime))}
```

Comment:
Not really important, but given that you're taking `log10` of the `expTime`, pretty much any value other than 10s would be better for the exposure time here, I think.

Reply:
Sure, no problem.